lib/rules/no-ember-data-global-imports.js

```diff
@@ -0,0 +1,165 @@
+'use strict';
+
+const MAPPINGS = require('@ember-data/rfc395-data');
+const { buildFix, buildMessage } = require('../utils/new-module');
+
+/**
+ * This function returns an object like this:
+ * {
+ *   'ember-data/model': {
+ *     default: ['@ember-data/model'],
+ *   },
+ * }
+ */
+function oldDataImportsReducer(acc, mapping) {
+  let obj = Object.create(null);
+  obj[mapping.export] = [mapping.replacement.module, mapping.replacement.export];
+  acc[mapping.module] = obj; // eslint-disable-line no-param-reassign
+
+  return acc;
+}
+
+const OLD_DATA_IMPORTS = MAPPINGS.reduce(oldDataImportsReducer, Object.create(null));
+
+/**
+ * This function returns an object like this:
+ * {
+ *   "DS.Model": {
+ *     global: "DS.Model",
+ *     export: "default",
+ *     localName: "Model",
+ *     module: "@ember-data/model" <- replacement
+ *   }
+ * }
+ */
+function globalsReducer(acc, mapping) {
+  const { global } = mapping;
+  if (global in acc) {
+    return acc;
+  }
+
+  let obj = {
+    global,
+    export: mapping.replacement.export,
+    localName: mapping.localName,
+    module: mapping.replacement.module,
+  };
+
+  acc[global] = obj;
+
+  return acc;
+}
+
+const GLOBALS = MAPPINGS.reduce(globalsReducer, Object.create(null));
+
+//------------------------------------------------------------------------------
+// Rule Definition - Use "Ember Data Packages" from Ember RFC #395
+//------------------------------------------------------------------------------
+
+module.exports = {
+  meta: {
+    docs: {
+      description: 'Use "Ember Data Packages" from Ember RFC #395',
+      category: 'Best Practices',
```

Comment:
It might be a good time for us to create a new `Ember Data` category. This rule along with the existing `no-empty-attrs` could go into it. If you make this change, remember to run `yarn update` afterward so the README gets updated.

Reply:
new category created

Reply:
Also I’ve addressed all your comments!